ivre/xmlnmap.py

```diff
@@ -199,6 +199,33 @@ def ignore_script(script):
     return False
 
 
+def cpe2dict(cpe_str):
+    """Helper function to parse CPEs. This is a very partial/simple parser."""
+    # Remove prefix
+    if len(cpe_str) < 5 or cpe_str[:5] != "cpe:/":
```

Comment:
Here, you can use `not cpe_str.startswith` instead of `len(cpe_str) < 5 or cpe_str[:5] != "cpe:/"`